index.js

```diff
@@ -353,5 +364,23 @@ formMobile.addEventListener('submit', (event) => {
     event.preventDefault();
   } else {
     messageMob.textContent = '';
+    const formDataMobile = {
+      name: mobileName.value,
+      email: emailMob.value,
+      text: mobileText.value,
+    };
+    const stringifyFormDataMobile = JSON.stringify(formDataMobile);
+    localStorage.setItem('mobileData', stringifyFormDataMobile);
   }
 });
+
+const storedMobileData = JSON.parse(localStorage.getItem('mobileData'));
+
+const storedDesktopData = JSON.parse(localStorage.getItem('desktopData'));
+
+mobileName.value = storedMobileData.name;
+emailMob.value = storedMobileData.email;
+mobileText.value = storedMobileData.text;
+desktopName.value = storedDesktopData.name;
+email.value = storedDesktopData.email;
+desktopText.value = storedDesktopData.text;
```

Comment:
- [ ] This is close as well but the form is pre-filled with the fetched data from `local storage` only when the page loads 🔃.
- **Hint:** You'll have to use the `window.onload` feature. 😉

Reply:
Issue resolved. Thank you.